apps/src/appMain.js

```diff
@@ -42,6 +44,24 @@ module.exports = function(app, levels, options) {
           options.level.levelBuilderRecommendedBlocks);
     }
 
+    if (marked && options.level.authoredHints) {
+      var hints;
+      try {
+        hints = JSON.parse(options.level.authoredHints);
+      } catch (e) {
+        hints = [];
+      }
+      level.authoredHints = hints.map(function (hint) {
+        return {
+          content: marked(hint.hint_markdown),
+          hintId: hint.hint_id,
+          hintClass: hint.hint_class,
+          hintType: hint.hint_type,
+          alreadySeen: false
+        };
+      });
+    }
```

Comment:
I wonder if this logic would better belong in authoredHintUtils? I.e. we do something like
```
level.authoredHints = authorizedHintUtils.generateAuthoredHints(options.level.authoredHints)
```
Also, what's the difference here between `options.level.authoredHints` and `level.authoredHints`? Does it end up just being the same variable but we have a preprocessed and postprecessed form?